src/main/resources/translations/Messages_nl.properties

```diff
@@ -0,0 +1,165 @@
+# Translators: Please only translate text. Don't change the prefix, colours or formatting.
+# Translators should also supply aliases that match translated versions of the commands
+
+# Minecraft colours may be used in this file for some messages
+# You may use an and-code for these - that is, colour codes beginning with the ampersand "&"
+# Any tokens "%s" are required and will be replaced by the plugin with context-sensitive information - do not remove these
+
+# This is the Dutch translation for archblock.
+# Dutch: Dit is de nederlandse vertaling van deze plugin.
+
+### GENERAL MESSAGES ###
+
+# This is put on the start of some messages
+# Please put a default colour at the end, it's all too easy to forget colours in messages
+message_prefix = &d[&6ArchBlock&d]&f
+
+command_no_permission = &cJe hebt geen toestemming om deze commando te gebruiken!
+ownership_blocks_changed = &9Eigennaarschap blockken van: &c%s&9 is verandert.
+player_only = Deze commando kan alleen in-game.
+player_or_console_only = Deze commando kan in-game of in de console.
+task_already_running = &cEr is al een taak bezig! even wachten aub.
+unable_to_update_block_owner = Kan blockken niet updaten bij: %s(%s, %s, %s): %s
+unknown_player = &cOnbekende speler: &b%s
+unknown_world = &cOnbekende wereld: &b%s
+
+### COMMAND ALIASES ###
+# Command aliases may be split with a semicolon ";"
+# You can leave any of these commented out if you don't want to use them
+# Translators should provide localized command names as aliases
+
```

Comment:
Are you going to do some command aliases? I feel like translating them would be good too